conftest.py

```diff
@@ -122,3 +123,36 @@ def pytest_configure(config):
     #
     # Somehow the default database is always in memory, though.
     settings.DATABASES['transient']['TEST_NAME'] = ':memory:'
+
+
+@pytest.yield_fixture()
+def installed_packages_catalog():
+    with mock.patch('ideascube.serveradmin.catalog.Catalog', spec=True) as MockClass:
```

Comment:
This is not exactly the same :)
The mocker.path if a wrapper around mock.path (or unittest.path, it depends of the python version. It unittest.path in our case)

One important change is that it start/stop the mock automatically.
But if we use the `with` syntax, the mock is then start/stop a second time.
I don't know if it works (and I don't want to know)

Another change : mocker do not have all the attribute of mock. (ie NonCallableMock)